docs/vs-2015/install/create-an-offline-installation-of-visual-studio.md

```diff
@@ -81,4 +81,4 @@ This page describes how to install Visual Studio 2015 when you are not connected
 |        Turkish        | http://go.microsoft.com/fwlink/?LinkID=785882&clcid=0x41F |
 
 ## See Also
- [Install Visual Studio]()
\ No newline at end of file
+ [Install Visual Studio](../install/install-visual-studio.md)
```

Comment:
The page for which you've suggested an edit is in our Visual Studio 2015 library, so the link must be "../install/install-visual-studio-2015.md"

Reply:
Please fix per my previous comment; thanks. :)

Reply:
Fixed! I noticed when I mad the change that it was a redirect file, but i figured that might have been fine.